extensions/amp-story-subscription/0.1/amp-story-subscription.js

```diff
@@ -0,0 +1,35 @@
+import {CSS} from '../../../build/amp-story-subscription-0.1.css';
+import {Layout_Enum} from '#core/dom/layout';
+import {htmlFor} from '#core/dom/static-template';
+
+const TAG = 'amp-story-subscription';
+
+// Create a paywall dialog element that have required attributes to be able to be
+// rendered by amp-subscriptions.
+// TODO: complete the rest of paywall dialog UI based on the publisher-provided attributes.
```

Comment:
Do we have an issue tracking the implementation of this feature? If not it may be a good idea. We can then tag it in todos.

Reply:
I can tag the I2I. Is there any special grammar to tag issue in the comment? If not, I would just include a link to I2I in the comment.

Reply:
yeah just the issue number is fine i.e. `TODO(#123): do this thing`. Sometimes I like to make a separate issue specifically tracking the steps needed for implementation but up to you.

Reply:
Done. Let me start by making everything tied to the I2I since there's no major discussion there. Thanks for the suggestion!